src/components/VideosLayout/VideoMiniature/VideoMiniature.component.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Card from '@material-ui/core/Card';
+import CardActionArea from '@material-ui/core/CardActionArea';
+import CardContent from '@material-ui/core/CardContent';
+import CardMedia from '@material-ui/core/CardMedia';
+import Typography from '@material-ui/core/Typography';
+import Tooltip from '@material-ui/core/Tooltip';
+
+const useStyles = makeStyles({
+  root: {
+    maxWidth: 410,
+  },
+  media: {
+    height: 150,
+  },
+});
+
+export const VideoMiniature = ( { video } ) => {
```

Comment:
Lo mismo que con las `classes` si lo único que vas a usar de tu objeto `video` es el `snippet` mejor destructúralo.

```suggestion
export const VideoMiniature = ( { video: { snippet } } ) => {
```

Solo ten en cuenta que al hacer esto tu parámetro `video` se vuelve inaccesible, debido a que estás renombrándolo como la destructuración del `snippet`. En este caso no tendrías problema porque no usas el objeto tal cual, pero de ser necesario podrías mejor destructurarlo dentro de la función, en su propia línea, así:

```suggestion
export const VideoMiniature = ( { video } ) => {
    const { snippet } = video.
```

Reply:
x2

Reply:
Si